### PR TITLE
chore: release v4

### PR DESCRIPTION
**Pull request by fzipi**
- update version across all files
- update project naming to `OWASP CRS`

### CODE REVIEW

**Comment by dune73** [comment]
You do not want to rename the project to CRS in the files?

**Comment by dune73** [comment]
You need to adjust the LICENSE copyright year too.

**Comment by dune73** [comment]
Otherwise, everything looks OK. Happy to approve and merge after fix.

**Review by dune73 — DISMISSED** [review]

File: rules/REQUEST-901-INITIALIZATION.conf [hunk 1]
```diff
@@ -58,8 +58,8 @@ SecRule &TX:crs_setup_version "@eq 0" \
     status:500,\
     log,\
     auditlog,\
-    msg:'ModSecurity Core Rule Set is deployed without configuration! Please copy the crs-setup.conf.example template to crs-setup.conf, and include the crs-setup.conf file in your webserver configuration before including the CRS rules. See the INSTALL file in the CRS directory for detailed instructions',\
-    ver:'OWASP_CRS/4.0.0-rc2',\
+    msg:'ModSecurity CRS is deployed without configuration! Please copy the crs-setup.conf.example template to crs-setup.conf, and include the crs-setup.conf file in your webserver configuration before including the CRS rules. See the INSTALL file in the CRS directory for detailed instructions',\
```
Comment (M4tteoP) [inline rules/REQUEST-901-INITIALIZATION.conf:61]: nit: just CRS?
```suggestion
    msg:'CRS is deployed without configuration! Please copy the crs-setup.conf.example template to crs-setup.conf, and include the crs-setup.conf file in your webserver configuration before including the CRS rules. See the INSTALL file in the CRS directory for detailed instructions',\
```

**Comment by dune73** [comment]
Nice catch with the `ver` tag. :)

And thanks for the hint @M4tteoP.

**Review by dune73 — DISMISSED** [review]

**Review by dune73 — APPROVED** [review]

**Pull request merged**